libclc/amdgcn-amdhsa/libspirv/workitem/get_local_size.cl

```diff
@@ -24,16 +24,16 @@ CONST_AS char * __clc_amdgcn_dispatch_ptr(void) __asm("llvm.amdgcn.dispatch.ptr"
 #endif
 
 _CLC_DEF _CLC_OVERLOAD size_t __spirv_WorkgroupSize_x() {
-    CONST_AS uint * ptr = (CONST_AS uint *) __dispatch_ptr();
-    return ptr[1] & 0xffffu;
+    CONST_AS ushort * ptr = (CONST_AS uint *) __dispatch_ptr();
```

Comment:
```suggestion
    CONST_AS ushort * ptr = (CONST_AS ushort *) __dispatch_ptr();
```

For libclc reviewers, the change in that file is mimic what clang codegen do https://github.com/intel/llvm/blob/sycl/clang/lib/CodeGen/CGBuiltin.cpp#L17614 (and simplifies support in kernel fusion)

Reply:
This kind of information should be in a comment, not in a PR comment. Otherwise we have a PR validation for a write-only code at the end... :-)